qiita_pet/templates/private_studies.html

```diff
@@ -3,13 +3,20 @@
 <link rel="stylesheet" href="/static/vendor/css/jquery.dataTables.css" type="text/css">
 
 <script src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<script src="/static/vendor/js/jquery.dataTables.plugin.natural.js"></script>
 
 <script type="text/javascript">
 var current_study;
 
 $(document).ready(function() {
-    $('#user-studies-table').dataTable({"order": [[ 1, "asc" ]]});
-    $('#shared-studies-table').dataTable({"order": [[ 1, "asc" ]]});
+        $('#user-studies-table').dataTable({
+            order: [[ 1, "asc" ]],
```

Comment:
This 1 has to be changed to a 0. Such that the function sorts using the study title when it the table is first loaded? Same for the rest of the the occurrences of this index.